src/v/config/seed_server.h

```diff
@@ -21,6 +21,9 @@
 namespace config {
 struct seed_server {
     unresolved_address addr;
+
+    bool operator==(const seed_server&) const = default;
+    bool operator!=(const seed_server&) const = default;
```

Comment:
Minor nit: The second is automagic when you specify the first. Also, I'm fascinated to try spaceship operator `<=>`  one day.